File: backend/model/credit.py
from model.customer import Customer

class Credit:
    def __init__(self,
                    property_url_original: str = None,
                    customer: Customer = None,
                    amount_requested: float = None,
                    comments: str = None,
                    property_url_modified: str = None,
                    format_credit_url: str = None,
                    credit_id: int = None):
        self.property_url_original = property_url_original
        self.customer = customer
        self.amount_requested = amount_requested
        self.comments = comments
        self.property_url_modified = property_url_modified
        self.format_credit_url = format_credit_url 
        self.comments = comments
        self.credit_id = credit_id

File: worker/visaji/Visaji/Models/Credit.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Visaji.Models
{
    [Table("credits")]
    public class Credit
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Column("credit_id")]
        public int Id { get; set; }

        [Column("upload_url_path")]
        public string UploadUrlPath { get; set; }
        
        [Column("property_url_original")]
        public string PropertyUrlOriginal { get; set; }

        [Column("property_url_modified")]
        public string PropertyUrlModified { get; set; }

        [Column("format_credit_url")]
        public string FormatCreditUrl { get; set; }
        
        [Column("amount_requested", TypeName = "decimal(18,0)")]
        public decimal AmmountRequested { get; set; }

        [Column("comments")]
        public string Comments { get; set; }

        [ForeignKey("Customer")]
        [Column("customer_id")]
        public int CustomerId { get; set; }

        public Customer Customer { get; set; }
    }
}
